### DISCUSSION TITLE
Icon request: goat

### DISCUSSION THREAD

**Discussion by Crissov** [FortAwesome/Font-Awesome#21164] [Icon Requests]
In order to fully support all local variants of Asian zodiac signs #5927 and European zodiac signs #2831, a Goat would be necessary. 

🐐 U+1F410 #222

**Comment by WillGoldstein** [comment]
Hoping this gets made!!

**Comment by kieranbehn** [comment]
+1

**Comment by gloversure-lwv** [comment]
I really, really, really want a goat. And I don't even like goats.

**Comment by gloversure-kbu** [comment]
Yes please!

**Comment by Klarence** [comment]
This icon could be the Greatest Of All Time!

Okay, that may be an exaggeration. +1 for me too.

**Comment by panchxyz** [comment]
Where is the goat?

**Comment by hsvandrew** [comment]
+1 - like the sheep, cow etc icons

**Comment by walapa-nl** [comment]
Still no goat :-(

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
